### PR TITLE
sdk: Cluster Alert Functions

### PR DESCRIPTION
**Pull request by lpabon**
**What this PR does / why we need it**:
Provides Alert functions for the SDK

**Which issue(s) this PR fixes** (optional)
Part of #407 

**Special notes for your reviewer**:
Changed SDK API functions only to ClusterAlertXXX to keep them all using the same style as the rest of the API calls.

### CODE REVIEW

File: api/api.proto [hunk 1]
```diff
@@ -654,19 +644,76 @@ message StorageCluster {
   repeated StorageNode nodes = 4;
 }
 
-// Enumerate
+service OpenStorageCluster {
+	// Enumerate lists all the nodes in the cluster.
+	rpc Enumerate(ClusterEnumerateRequest)
```
Comment (sangleganesh) [inline api/api.proto]: nit: indentation

File: api/server/sdk/cluster_test.go [hunk 2]
```diff
@@ -291,3 +293,105 @@ func TestSdkInspectIdNotPassed(t *testing.T) {
 	assert.Equal(t, serverError.Code(), codes.InvalidArgument)
 	assert.Contains(t, serverError.Message(), "Node")
 }
+
+func TestSdkAlertEnumerate(t *testing.T) {
```
Comment (sangleganesh) [inline api/server/sdk/cluster_test.go]: what is this test testing ?

Comment (lpabon) [reply api/server/sdk/cluster_test.go]: It is testing that the grpc connection goes to the server, then the server calls the interface, and returns the expected value.

File: api/api.proto (hunk 1 above)
Comment (lpabon) [reply api/api.proto]: Weird, I'm not sure what is going on there with the tabs. I'll check it.

File: api/api.proto [hunk 3]
```diff
@@ -654,19 +644,76 @@ message StorageCluster {
...
+}
+
+message ClusterAlertClearResponse {
+}
+
+message ClusterAlertEraseRequest {
+  // Type of resource (required)
+  ResourceType resource = 1;
+  // Id of alert as returned by ClusterEnumerateAlertResponse (required) 
+	int64 alert_id = 2;
```
Comment (adityadani) [inline api/api.proto]: nit: indentation
There are multiple indentation issues in this file.

File: api/api.proto [hunk 4]
```diff
@@ -654,19 +644,76 @@ message StorageCluster {
...
+	// Enumerate lists all the nodes in the cluster.
+	rpc Enumerate(ClusterEnumerateRequest)
+		returns (ClusterEnumerateResponse) {}
+
+	// Inspect the node given a UUID.
+	rpc Inspect(ClusterInspectRequest)
+    returns (ClusterInspectResponse) {}
+    
+  // Get a list of alerts from the storage cluster
+  rpc AlertEnumerate(ClusterAlertEnumerateRequest)
```
Comment (adityadani) [inline api/api.proto]: Is this a RPC standard to name it as AlertEnumerate? IMO EnumerateAlert sounds more intuitive.

File: api/api.proto [hunk 5]
```diff
@@ -654,19 +644,76 @@ message StorageCluster {
...
 }
 
-// Enumerate
+service OpenStorageCluster {
+	// Enumerate lists all the nodes in the cluster.
+	rpc Enumerate(ClusterEnumerateRequest)
+		returns (ClusterEnumerateResponse) {}
+
+	// Inspect the node given a UUID.
+	rpc Inspect(ClusterInspectRequest)
```
Comment (adityadani) [inline api/api.proto]: nit: maybe rename it to InspectNode

File: api/api.proto (hunk 4 above)
Comment (lpabon) [reply api/api.proto]: Trying to standardize on reverse object naming style ---> direction. LIke Cluster->Alert->Enumerate.. Or AAA->BBB->CCCC for the API

File: api/api.proto (hunk 5 above)
Comment (lpabon) [reply api/api.proto]: Good one. I will make that change.

**Pull request merged**